addons/planner_project/static/src/less/planner_project.less

```diff
@@ -0,0 +1,62 @@
+
+#PlannerModal {
```

Comment:
Since the #PlannerModal id is in a js qweb template, put the new `planner_xxxxx` class with the `oe_planner` class in the view directly to not add it with javascript ;)